concepticondata/conceptlists/Huang-1992-1820.tsv

```diff
@@ -1486,7 +1486,7 @@ Huang-1992-1820-1484	1484	idle, be / unoccupied	空闲			v.	idle, be / unoccupie
 Huang-1992-1820-1485	1485	cry	哭			v.	cry
 Huang-1992-1820-1486	1486	sleepy, be	困(~倦)	3620	SLEEPY	v.	sleepy, be
 Huang-1992-1820-1487	1487	pull / tug	拉	1455	PULL	v.	pull / tug
-Huang-1992-1820-1488	1488	empty the bowels	拉屎	676	SHIT (DEFECATE)	v.	empty the bowels
+Huang-1992-1820-1488	1488	empty the bowels	拉屎			v.	empty the bowels
```

Comment:
I'd say "empty the bowels" is a polite way of saying "shit" but it is the same process.

Reply:
Ah, thanks for catching that - that unmapping was just a mistake due to having to manually integrate some other update and overlooking this one.